include/zephyr/bluetooth/conn.h

```diff
@@ -960,6 +1087,32 @@ struct bt_conn_cb {
 			      const struct bt_df_conn_iq_samples_report *iq_report);
 #endif /* CONFIG_BT_DF_CONNECTION_CTE_RX */
 
+#if defined(CONFIG_BT_CTLR_LE_POWER_CONTROL)
```

Comment:
Define a Host Kconfig, because host only applications will not use controller Kconfig options

Reply:
I will define `BT_POWER_CONTROL` and `BT_PATH_LOSS_MONITORING ` at the end of `BT_CONN` dependent configs.

I will remove `BT_CTLR_LE_POWER_CONTROL_SUPPORT`,  so hci command / feature support will be set with `BT_POWER_CONTROL` and `BT_PATH_LOSS_MONITORING `accordingly.